docs/api/sql/GeoSparkSQL-Function.md

```diff
@@ -269,6 +269,18 @@ SELECT ST_GeometryType(polygondf.countyshape)
 FROM polygondf
 ````
 
+## ST_LineMerge
+
+Introduction: Returns a LineString formed by sewing together the constituent line work of a MULTILINESTRING.
+
+Note: Only works for MULTILINESTRING. Using other geometry will return a GEOMETRYCOLLECTION EMPTY. If the MultiLineString can't be merged, the original MULTILINESTRING is returned.
+
```

Comment:
Please add - Since: v1.3.2

Reply:
Ups... this must have gone missing when I was fixing the conflicts. It's fixed now.